homeassistant/components/unifi/controller.py

```diff
@@ -251,30 +262,9 @@ async def initialize(self) -> None:
         assert self.config_entry.unique_id is not None
         self.is_admin = self.api.sites[self.config_entry.unique_id].role == "admin"
 
-        # Restore clients that are not a part of active clients list.
-        entity_registry = er.async_get(self.hass)
-        for entry in async_entries_for_config_entry(
-            entity_registry, self.config_entry.entry_id
-        ):
-            if entry.domain == Platform.DEVICE_TRACKER:
-                mac = entry.unique_id.split("-", 1)[0]
-            elif entry.domain == Platform.SWITCH and entry.unique_id.startswith(
-                BLOCK_SWITCH
-            ):
-                mac = entry.unique_id.split("-", 1)[1]
-            else:
-                continue
-
-            if mac in self.api.clients or mac not in self.api.clients_all:
-                continue
-
-            client = self.api.clients_all[mac]
-            self.api.clients.process_raw([dict(client.raw)])
-            LOGGER.debug(
-                "Restore disconnected client %s (%s)",
-                entry.entity_id,
-                client.mac,
-            )
```

Comment:
Is this code no longer needed or did we only care about it for blocked clients?

Reply:
The logic was duplicated in how to retrieve blocked clients but also the device tracker side of it was more relevant when all entities was always created, so now Im trying to clean up old things not being as beneficial as before. 

Moving forward with https://github.com/home-assistant/core/pull/99483 the aim to be more explicit in what clients are wanted within the integration